checks.d/kafka_consumer.py

```diff
@@ -73,56 +127,75 @@ def check(self, instance):
         kafka_conn = KafkaClient(kafka_host_ports, timeout=self.kafka_timeout)
 
         try:
-            # Query Kafka for the broker offsets
+            # Query Kafka to find the latest message committed to the broker
+            # so we can calculate consumer lag
             broker_offsets = {}
-            for topic, partitions in topics.items():
+            for topic, partitions in tracked_topic_partitions.iteritems():
                 offset_responses = kafka_conn.send_offset_request([
-                    OffsetRequest(topic, p, -1, 1) for p in partitions])
+                    OffsetRequestPayload(topic, p, -1, 1) for p in partitions])
 
                 for resp in offset_responses:
-                    broker_offsets[(resp.topic, resp.partition)] = resp.offsets[0]
+                    key = (resp.topic, resp.partition)
```

Comment:
I did it for consistency with L#113:

    key = (consumer_group, topic, partition)

Would you prefer I collapse both of them, collapse one, or leave both?